src/applications/pensions/config/chapters/01-applicant-information/applicantInformation.js

```diff
@@ -79,7 +81,8 @@ export default {
       },
       veteranFullName: fullNameSchema,
       veteranSocialSecurityNumber: ssnSchema,
-      vaClaimsHistory: yesNoSchema,
+      vaClaimsHistory,
```

Comment:
The common definitions become $refs in the vets-json-schema dist, but they get turned back into the full definitions when passing them to the components (we also import the relevant definitions to allow that). That all gets handled by the form system, as long as we pass the correct definitions to the form config https://github.com/department-of-veterans-affairs/vets-website/blob/39e38ef10d72d94b33d18e699a1495b1ae8bf68d/src/applications/pensions/config/form.js#L33